packages/usa-range/src/styles/_usa-range.scss

```diff
@@ -32,12 +32,24 @@
   border: units(1px) solid color("base");
 }
 
-.usa-range {
-  @extend %block-input-general;
-  @extend %block-input-styles;
+@mixin range-thumb--disabled {
+  @include range-thumb;
+  @include u-disabled;
+}
+
+@mixin range-track--disabled {
+  @include range-track;
+  @include u-disabled;
+}
+
+@mixin range-ms-fill--disabled {
+  @include range-ms-fill;
```

Comment:
Is everything from this mixin needed? It looks like some of the styles set with `range-ms-fill` end up being overwritten by `u-disabled`.

If we're also using it below in `-ms-fill-upper` why not consider something like breaking out the required styles from `range-track to something like `range-track-base` and importing that in the places where you **only** need those base styles?

Reply:
It looks like I was able to remove these disabled styles and simply add the `u-disabled` mixin once `usa-range` is disabled/aria-disabled

Resolved in [a753027](https://github.com/uswds/uswds/pull/5063/commits/a753027d552ef32bc708778b380c4c3079ed6d17)